app/assets/stylesheets/_variables.scss

```diff
@@ -26,29 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #DDD;
 
-// Background Color
-$base-background-color: white;
-
 // Font Colors
 $base-font-color: $dark-gray;
-$base-accent-color: $blue;
-
-// Link Colors
-$base-link-color: $base-accent-color;
-$hover-link-color: darken($base-accent-color, 15);
```

Comment:
I sure think we should keep these in. It's important to have a consistent relationship between the action color and its alt/hover state. Perhaps we should add `$base-action-color` and `$hover-action-color`. I feel silly when i'm styling buttons and writing `&:hover { background-color: shade($action-color, some-arbitrary-value)}`